corehq/apps/userreports/pillow.py

```diff
@@ -45,6 +46,7 @@ def timed(*args, **kw):
                 table.config._id, doc['_id'], seconds
             )
             pillow_logging.warning(message)
+            _slow_ucr_assert(seconds < 5, message)
```

Comment:
This doesn't do much other than alert me that a particular ucr is slow. I still need to go in and see why it's slow, so I don't think sentry would have much gain